pkg/remote/git.go

```diff
@@ -116,6 +121,9 @@ func (g gitRemoteLoader) Load(ctx context.Context, path string) (string, error)
 	if stat.IsDir() {
 		local, err = findFile(cli.DefaultFileNames, local)
 	}
+	if progress.Mode != progress.ModeQuiet && progress.Mode != progress.ModeJSON {
+		_, _ = fmt.Fprintf(g.dockerCli.Out(), "Your compose stack %q is stored in %q\n", strings.TrimPrefix(path, prefix), local)
```

Comment:
this message will print for every Git remote loaded by compose, including those involved by `includes`
Maybe better load project, then check if `project.ComposeFiles[0]` is remote _then_ print result of `Dir(".")` as location where remote stack has been stored on disk